client/app/kubernetes/nodes/kubernetes-nodes.html

```diff
@@ -0,0 +1,54 @@
+<div data-ng-if="$ctrl.loading" class="text-center">
+    <oui-spinner data-size="m"></oui-spinner>
+</div>
+<div data-ng-if="!$ctrl.loading">
+    <cui-message-container data-messages="$ctrl.messages"></cui-message-container>
+    <div data-ng-if="$ctrl.nodes">
+        <div class="mb-3">
+            <span data-translate="kube_nodes_description_project_1"></span>
+            <a class="oui-link"
+               data-ng-bind=":: $ctrl.project.description"
+               data-ui-sref="iaas.pci-project.compute({ projectId: $ctrl.project.project_id })"></a>
+            <span data-translate="kube_nodes_description_project_2"></span>
+        </div>
+        <oui-datagrid data-rows="$ctrl.nodes" data-row-loader="$ctrl.getAssociatedFlavor($row)">
+            <oui-column data-title=":: 'kube_nodes_node' | translate"
+                        data-property="id"
+                        data-type="string"
+                        data-searchable
+                        data-filterable></oui-column>
+            <oui-column data-title=":: 'kube_nodes_flavor' | translate"
+                        data-property="formattedFlavor"
+                        data-type="string"
+                        data-searchable
+                        data-filterable>
+            </oui-column>
+            <oui-column data-title=":: 'kube_service_cluster_status' | translate"
+                        data-property="status">
+                <span class="oui-status"
+                      data-ng-class="{
+                        'oui-status_success' : $row.status === 'READY',
+                        'oui-status_warning' : $row.status === 'INSTALLING' || $row.status === 'DELETING',
+                        'oui-status_error' : $row.status === 'ERROR'
+                        }"
+                      data-translate="{{:: 'kube_service_cluster_status_' + $row.status }}">
+                </span>
+            </oui-column>
+            <oui-action-menu data-disabled="$row.status === 'INSTALLING' || $row.status === 'DELETING'"  data-align="end" data-compact>
+                <oui-action-menu-item data-text="{{:: 'common_delete' | translate }}" data-on-click="$ctrl.confirmNodeDeletion($row.id)"></oui-action-menu-item>
+            </oui-action-menu>
+            <extra-top>
```

Comment:
`<extra-top>` component encourage us to use `<oui-action-menu>` component.

example for the documentation:
```html
  <extra-top>
    <oui-action-menu text="Actions" aria-label="Server: actions" align="start">
      <oui-action-menu-item text="Add person" aria-label="Persons: add item"></oui-action-menu-item>
      <oui-action-menu-item text="Other action" aria-label="Persons: other action"></oui-action-menu-item>
    </oui-action-menu>
  </extra-top>
```
Can you tell me if that's fit your needs?

Reply:
For 2 actions or more, use an action menu (guidelines).